### PR TITLE
68 throw error on the backend instead of an object with a status and message

### PR DESCRIPTION
**Pull request by jasoncavanaugh**

### CODE REVIEW

**Comment by jasoncavanaugh** [comment]
This originally started out as an effort to make error handling on the backend more consistent, and then it kinda turned into a bit of a refactor. Not totally sure if it was worth it. Will definitely need to do some manual testing to make sure everything still works as expected.

File: server/routes/events.ts [hunk 1]
```diff
@@ -152,20 +153,7 @@ router.route("/api/events").get(
...
-    });
-    if (!resp.ok) {
-      throw {
-        message: AIRTABLE_ERROR_MESSAGE,
-        status: resp.status,
-      };
-    }
-    const futureEventsAirtableResp =
-      (await resp.json()) as AirtableResponse<Event>;
+    const futureEventsAirtableResp = await airtableGET<Event>({ url: url });
```
Comment (mattsahn) [inline server/routes/events.ts:156]: These are really nice as one line here now.

**Comment by mattsahn** [comment]
> 

Nice work. I'm a fan of the refactor moving all the repetitive auth/http stuff to one place. I clicked around the railway PR link and it all looks to be working well. I love that railway feature.

**Review by mattsahn — APPROVED** [review]
LGTM

**Pull request merged**